### PR TITLE
Adding Theme Features / Crisis Themes

### PR DESCRIPTION
**Pull request by TerallianHub**
Added Crisis Refugee Theme
Added Theme Features Compendium

### CODE REVIEW

**Comment by LebombJames** [comment]
I'm not a fan of including the Theme's name in the title of the Theme Feature. It should only be the Feature's name. E.g "Care for Others"

**Comment by TerallianHub** [comment]
> I'm not a fan of including the Theme's name in the title of the Theme Feature. It should only be the Feature's name. E.g "Care for Others" (I suppose Theme Knowledge can be exempt of this, so we don't have loads of features all called Theme Knowledge)

I'll keep it just for Theme Knowledge then and remove it from the rest. Should we maybe format it like Weapon Specialization, ie: "Theme Knowledge (Theme Name)"?

**Comment by LebombJames** [comment]
That's a good idea, yeah

**Comment by TerallianHub** [comment]
Latest push should have: Crisis Refugee, Drift Crashed, Opportunist, Planar Traveler, Spectra Scion and Void Nomad, as well as their Theme Features.

**Comment by LebombJames** [comment]
Git seems to think you've renamed and modified some files when creating new ones. Can you verify the files that it thinks you have renamed still exist? Might be worth running `cook` just to refresh your JSONs

**Comment by TerallianHub** [comment]
> 

I've given them another look over and it should be all correct from what I can tell. I did notice in running the cook I'd missed two description links so included those now.

**Comment by ian612** [comment]
I've made a few updates to this to get it ready for merging. I'm going to re-review the changed items to make sure everything's good, and then merge.

File: src/packs/theme-features.db
Comment (danimrath) [inline src/packs/theme-features.db:1]: Is this supposed to be empty?

**Review by ian612 — APPROVED** [review]

**Pull request merged**